conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/ethereum/EthereumDataOperationsTest.scala

```diff
@@ -0,0 +1,253 @@
+package tech.cryptonomic.conseil.api.routes.platform.data.ethereum
+
+import java.sql.Timestamp
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.OptionValues
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import slick.jdbc.PostgresProfile.api._
+import tech.cryptonomic.conseil.api.EthereumInMemoryDatabaseSetup
+import tech.cryptonomic.conseil.common.ethereum.EthereumTypes.EthereumBlockHash
+import tech.cryptonomic.conseil.common.ethereum.Tables
+import tech.cryptonomic.conseil.common.ethereum.Tables._
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes.Query
+import tech.cryptonomic.conseil.common.testkit.InMemoryDatabase
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+class EthereumDataOperationsTest
+    extends AnyWordSpec
+    with Matchers
+    with InMemoryDatabase
+    with EthereumInMemoryDatabaseSetup
+    with ScalaFutures
+    with OptionValues
+    with LazyLogging
+    with IntegrationPatience
+    with EthereumDataOperationsTest.Fixtures {
+
+  "EthereumDataOperations" should {
+      val sut = new EthereumDataOperations("ethereum") {
+        override lazy val dbReadHandle = dbHandler
+      }
+
+      "return proper number of blocks, while fetching all blocks" in {
+        // given
+        dbHandler.run(Tables.Blocks ++= blocks).isReadyWithin(5.seconds) shouldBe true
+
+        whenReady(sut.fetchBlocks(Query.empty)) { result =>
+          result.value.size shouldBe 3
```

Comment:
I'm usually quite picky about this kind of things, so this is also a request for opinions:

Is it better to hardcode the number of expected values, or to reuse the fixture data to compute it, so it remains in sync with the fixture definition, whatever might happen? I.e. I'd choose

```scala
result.value.size shouldBe (blocks.size)
```

And the same holds for all the similar cases following this one

Reply:
Yea, I thought that it was better to have what you proposed for couple of years, but then I realized that changing fixtures had to big of an impact on rest of the tests. Meaning that it's wasn't my conscious decision that assertion changed. Since tests are really important and I like when I see that some conditions are changing explicitly in pull-requests (instead of implicit via fixture changes) I think it's better to have them explicitly mentioned. Even, when you are comparing the same stuff (like asking for specific `id` and checking if the same `id` has been returned, I would code them twice).